packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -273,7 +274,7 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     this._openMenu(shouldFocusOnContainer, shouldFocusOnMount);
   }
 
-  private _onRenderContent(tag: any, buttonProps: IButtonProps): JSX.Element {
+  private _onRenderContent(tag: any, buttonProps: IButtonProps & { ref?: any }): JSX.Element {
```

Comment:
Is there a built-in React interface for this?

Reply:
There is not